### PR TITLE
plotting normalized score

### PR DESCRIPTION
**Pull request by hnyu**

### CODE REVIEW

File: alf/utils/plot_tb_curves.py [hunk 1]
```diff
@@ -229,31 +249,99 @@ def y_label(self):
...
+                 dpi=100,
+                 linestyle='-',
+                 linewidth=2,
+                 alpha=0.3,
+                 bg_color='#EEEEEE',
+                 grid_color='#DEDEDE',
+                 legend_kwargs=dict(loc="best"),
                  title=None):
-        """
+        r"""
```
Comment (emailweixu) [inline alf/utils/plot_tb_curves.py:339]: Add simple description what the plot looks like.

File: alf/utils/plot_tb_curves.py [hunk 2]
```diff
@@ -229,31 +249,99 @@ def y_label(self):
         return "Success Rate"
 
 
+class MeanCurveGroupReader(object):
+    r"""Group several ``MeanCurveReader`` results. A ``MeanCurveGroupReader`` is
+    suitable for one method on multiple tasks, each task with multiple runs.
+    To aggregate across tasks, each task must be provided with a performance
+    range :math:`(y_0, y_1)` that will be used to normalize performance for that
+    task as :math:`\frac{y - y_0}{y_1 - y_0}`.
+    """
```
Comment (emailweixu) [inline alf/utils/plot_tb_curves.py:258]: perhaps also include an example of using this somewhere.

File: alf/utils/plot_tb_curves.py (hunk 1 above)
Comment (hnyu) [reply alf/utils/plot_tb_curves.py:339]: added

File: alf/utils/plot_tb_curves.py (hunk 2 above)
Comment (hnyu) [reply alf/utils/plot_tb_curves.py:258]: updated the example

**Review by emailweixu — APPROVED** [review]

**Pull request merged**